pkgs/development/python-modules/pytask/default.nix

```
@@ -0,0 +1,87 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  hatchling,
+  hatch-vcs,
+  attrs,
+  click,
+  click-default-group,
+  networkx,
+  optree,
+  packaging,
+  pluggy,
+  rich,
+  sqlalchemy,
+  universal-pathlib,
+  pytestCheckHook,
+  deepdiff,
+  nbmake,
+  pexpect,
+  pytest-cov,
+  pytest-xdist,
+  syrupy,
+  aiohttp,
+  git,
+}:
+buildPythonPackage rec {
+  pname = "pytask";
+  version = "0.5.1";
+  format = "pyproject";
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "pytask-dev";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-b+sS+l0Rp5bb8Dh6UBv3xHYTYKFp3dD5AuLqxB3n6Go=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+    hatch-vcs
+  ];
+
+  propagatedBuildInputs = [
+    attrs
+    click
+    click-default-group
+    networkx
+    optree
+    packaging
+    pluggy
+    rich
+    sqlalchemy
+    universal-pathlib
+  ];
```

```suggestion
  ] ++ lib.optionals (pythonOlder "3.11") [
    tomli
  ];
```
https://github.com/pytask-dev/pytask/blob/b569819abde7f8f1362f95712621be2ce2346c71/pyproject.toml#L33

---

Good catch, thanks!